torch/_inductor/ir.py

```diff
@@ -520,16 +539,16 @@ def get_workspace_size(self):  # type: ignore[no-untyped-def]
 class Loops(IRNode):
     device: torch.device
     dtype: torch.dtype
-    inner_fn: Callable[..., Any]
-    ranges: List[Expr]
+    inner_fn: Callable[..., OpsValue]
+    ranges: Sequence[_IntLike]
 
-    def get_unbacked_symbol_uses(self) -> OrderedSet[sympy.Symbol]:
+    def get_unbacked_symbol_uses(self) -> OrderedSet[Symbol]:
         return OrderedSet().union(
             *(free_unbacked_symbols(e) for e in self.ranges),
             self.inner_fn_free_unbacked_symbols(),
         )
 
-    def __str__(self, names=("ranges",)):  # type: ignore[no-untyped-def]
+    def __str__(self, names: Tuple[str] = ("ranges",)) -> str:  # type: ignore[no-untyped-def]
```

Comment:
Why still keep this comment?

Reply:
Actually, important nit. Does names always have to be a length 1 Tuple? Otherwise,
```suggestion
    def __str__(self, names: Tuple[str, ...] = ("ranges",)) -> str:  # type: ignore[no-untyped-def]
```
would be appropiate.